wiki/Manual.md

```diff
@@ -5032,7 +5032,7 @@ cancelOrders (ids, symbol = undefined, params = {})
 Parameters
 
 - **ids** (\[String\]) *required* Order ids (e.g. `1645807945000`)
-- **symbol** (String) Unified CCXT market symbol **required** on some exchanges (e.g. `"BTC/USDT"`)
```

Comment:
we're not saying anything different here 😅  required on some = optional in others